recipes/ugrid-checks/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ugrid-checks" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51db760aaaf26b43f8ff1f681a183e42d4a1f84bba8c5ee9bb24128c4acb4d00
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - ugrid-checker = ugrid_checks.cli:call_cli
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools >=45
+  run:
+    - python >=3.7
+    - netCDF4
+
+test:
+  imports:
+    - ugrid_checks 
+  commands:
+    - ugrid-checker --help
+
+about:
+  home: https://github.com/pp-mo/ugrid-checks 
```

Comment:
@bjlittle and @pp-mo I'd love to her from you how this compares to https://github.com/ioos/cc-plugin-ugrid. Lately, every time I see a duplicated effort like that, I feel like we failed as a community :disappointed:

Reply:
@ocefpaf Thanks so much... we literally just discovered this package the other day, which is indeed a shame.

That said, perhaps there is a potential to joint forces and collaborate somehow? :thinking:

Reply:
I'd love that. IOOS is not married to any package they develop. Is something better appears we try to focus our efforts on what the community is using. With that said, compliance-checker has a really large users base and it is by far the project with the largest bus fator in our repository. The plugins not so much though.

The main problem here is that we are committed to the plugin functionality and ugrid-tests is a stand-alone. it would be nice to know if our plugin can do everything you need, if not we can assess if it is worth fixing it to be what you need or if we should give up that particular plugin. Does that make sense? Also, we should move this to our issue page.